sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Models/NotificationHookModelTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.AI.MetricsAdvisor.Administration;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.MetricsAdvisor.Tests
+{
+    public class NotificationHookModelTests : MockClientTestBase
+    {
+        public NotificationHookModelTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private string UnknownHookContent => $@"
+        {{
+            ""hookId"": ""{FakeGuid}"",
+            ""hookName"": ""unknownHookName"",
+            ""hookType"": ""unknownType"",
+            ""externalLink"": ""https://fakeuri.com/"",
+            ""description"": ""unknown hook description"",
+            ""admins"": [
+              ""foo@contoso.com""
+            ]
+        }}
+        ";
+
+        [Test]
+        public async Task NotificationHookGetsUnknownHook()
```

Comment:
In this first test we're making sure we can parse a response from the service when we get back an unknown hook.